students/taekwon/users/views.py

```diff
@@ -48,3 +48,23 @@ def post(self, request):
             return JsonResponse({'message': 'SUCCESS'}, status=201)
         except KeyError:
             return JsonResponse({'message': 'KEY_ERROR'}, status=400)
+
+class LogInView(View):
+    def post(self, request):
+        try:
+            data        = json.loads(request.body)
+            email       = data['email']
+            password    = data['password']
+
+            if not User.objects.filter(email = email).exists():
+                return JsonResponse({'message': 'INVALID_USER'}, status=401)
+
+            if User.objects.filter(email = email).exists():
+                valid_user = User.objects.get(email = email)
```

Comment:
위의 if에서 존재 여부를 확인했기 때문에 다시 조건문에 넣을 필요는 없습니다.
```suggestion

            valid_user = User.objects.get(email = email)
```